float32 y_position
string player_name